### ISSUE TITLE
[Feature] Publish Schema Package

### ISSUE THREAD

**Issue by SelfhostedPro** [AmruthPillai/Reactive-Resume#2123]
### Is there an existing issue for this feature?

- [X] Yes, I have searched the existing issues and it doesn't exist.

### Feature Description

I would love to be able to automatically parse json resumes exported from Reactive-Resume but can't seem to find @reactive-resume/schema published anywhere. Is there anyway you'd be wiling to publish on npm?

**Comment by DemaPy** [comment]
Did you ask about this one?

> First, head over to your resume on Reactive Resume and scroll down to the “Export” section on the right panel. Here, you have two options, to export the resume as JSON, or as a PDF.

**Comment by SelfhostedPro** [comment]
Hi, yes, that’s not what it’s in reference to. I’m referring to the schema the app uses in order to parse and validate said resumes.

**Comment by lightyoruichi** [comment]
#2159 @SelfhostedPro this is what you're looking for.

**Comment by SelfhostedPro** [comment]
No, I’m asking them to consider publishing the schema library they’re using internally: https://github.com/AmruthPillai/Reactive-Resume/tree/main/libs/schema

**Comment by amruthpillai** [comment]
Closing this issue as I'm doing some clean up to move to a newer version of the project. I apologize in advance for the inconvenience if you find the need to reopen this issue.

Please read #2499 for more information about the migration.
